app/forms/openid_connect_authorize_form.rb

```diff
@@ -119,11 +132,11 @@ def success_redirect_uri
   end
 
   def ial_values
-    acr_values.filter { |acr| acr.include?('ial') || acr.include?('loa') }
+    IALS_BY_PRIORITY & acr_values
```

Comment:
(A/N): Using an array intersection in this order means that it will both remove unknown values and sort in the same order as the _first_ array in the intersection expression. This means nothing else should sort the ial/aal values.

Reply:
this is a neat solution!